UPDATING.md

```diff
@@ -105,6 +105,17 @@ default representation (`__repr__`).
 The old method is still works but can be abandoned at any time. The changes are intended to delete method
 that are rarely used.
 
+### Made BaseSensorOperator respect the trigger_rule of downstream tasks
+
+Previously, BaseSensorOperator with setting soft_fail=True becomes skipped itself
+and skips all its downstream tasks unconditionally when it fails.
+The point is it does not respect the trigger_rule of downstream tasks when it fails.
+In the new behavior, the trigger_rule of downstream tasks are respected.
+User can preserve/achieve the original behaviour by setting every downstream task to all_success,
+because downstream tasks with trigger_rule all_success (i.e. the default) are skipped
+when upstream task is skipped.
+
+
```

Comment:
Can you move that to the top when you rebase? :)

Reply:
I fixed it.